src/ducks/albums/components/AlbumToolbar.jsx

```diff
@@ -6,19 +6,24 @@ import { withRouter } from 'react-router'
 import { translate } from '../../../lib/I18n'
 
 import DeleteButton from '../../../components/DeleteButton'
+import ShareButton from '../../../components/ShareButton'
 import Alerter from '../../../components/Alerter'
 import Menu, { MenuButton, Item } from 'react-bosonic/lib/Menu'
 
 import { showSelectionBar } from '../../../actions'
 import { mustShowSelectionBar } from '../../../reducers'
-import { deleteAlbum } from '..'
+import { deleteAlbum, getAlbumShareLink } from '..'
 import DestroyConfirm from '../../../components/DestroyConfirm'
 import confirm from '../../../lib/confirm'
+import ShareModal from '../../../containers/ShareModal'
 
 import classNames from 'classnames'
 
-export const AlbumToolbar = ({ t, album, disabled = false, uploadPhotos, deleteAlbum, selectItems, params }) => (
+export const AlbumToolbar = ({ t, album, disabled = false, uploadPhotos, deleteAlbum, selectItems, shareAlbum }) => (
   <div className={styles['pho-toolbar']} role='toolbar'>
+    <ShareButton
```

Comment:
I do not understand what you mean! See below:

![screenshot at 09-24-25](https://cloud.githubusercontent.com/assets/701648/26190971/3e65cb60-3bac-11e7-938a-419c96fdcd53.png)
![screenshot at 09-24-34](https://cloud.githubusercontent.com/assets/701648/26190970/3e328f84-3bac-11e7-8801-1b22cc3111b0.png)
![screenshot at 09-24-40](https://cloud.githubusercontent.com/assets/701648/26190969/3e302ed8-3bac-11e7-995f-13974b862d9f.png)